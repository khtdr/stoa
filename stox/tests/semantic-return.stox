#!/usr/bin/env stox

{ // should't be allowed
  return;
}
