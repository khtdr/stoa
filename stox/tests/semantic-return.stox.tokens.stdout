LEFT_CURL[3,1]
RETURN[4,3]
SEMICOLON[4,9]
RIGHT_CURL[5,1]
